Update directx.md with link to DX Landing Page

JacquesvanRhynMSFT opened the pull request:
Added link to the DirectX Landing Page at the top of this readme to make sure all DX-related components and content link back to our new single source of truth.

JacquesvanRhynMSFT committed:
Update directx.md with link to DX Landing Page 

Added link to the DirectX Landing Page at the top of this readme to make sure all DX-related components and content link back to our new single source of truth.

PRMerger12 commented:
@JacquesvanRhynMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims committed:
Edits

Making this consistent with the similar link in the DWriteCore overview topic.

There's also one in the "Developer guide for redistributable version of XAudio 2.9" topic, which I feel would be better worded like the other two.

PRMerger18 commented:
@stevewhims : Thanks for your contribution! The author(s) have been notified to review your proposed change.
